### PR TITLE
add new logo in sidebar_header.ejs #420

### PR DESCRIPTION
**Pull request by marti1125**
please review

### CODE REVIEW

**Comment by marti1125** [comment]
@cvan  please review :+1:

**Comment by cvan** [comment]
@marti1125 this looks broken on the [Docs](http://localhost:4000/docs/0.5.0/introduction/)/secondary pages:

screen shot 2017-04-10 at 10 29 57 pm

File: themes/aframe/source/css/secondary.styl [hunk 1]
```diff
@@ -88,6 +88,10 @@ a
     +media--desktop()
         line-height 1.6
 
+.sponsor
+    background-color:  #ef2d5e
```
Comment (cvan) [inline themes/aframe/source/css/secondary.styl]: remove `:`

File: themes/aframe/source/css/secondary.styl [hunk 2]
```diff
@@ -88,6 +88,10 @@ a
...
+    position: relative    
+
```
Comment (cvan) [inline themes/aframe/source/css/secondary.styl]: I was able to fix the styles of this by doing this:

```diff
diff --git a/themes/aframe/source/css/_common.styl b/themes/aframe/source/css/_common.styl
index 6b3687c..f0016ce 100644
--- a/themes/aframe/source/css/_common.styl
+++ b/themes/aframe/source/css/_common.styl
@@ -227,8 +227,8 @@ h2, h3, h4
     .logo__wordmark
         margin-bottom 20px
 
-.sponsor {  
-  vertical-allign: middle;
+.sponsor {
+  vertical-align: middle;
   position: absolute;
   bottom: 0;
   right: 0;
@@ -248,9 +248,9 @@ a.mozilla-logo {
   opacity: 0.8;
   background: url(../images/mozilla-logo.svg) no-repeat;
   height: 24px;
-  width: 88px;
+  width: 84px;
   border: none;
-  position: relative;
+  margin-left: .25ch;
   top: 1px;
   transition: .15s opacity ease-in-out;
 }
diff --git a/themes/aframe/source/css/secondary.styl b/themes/aframe/source/css/secondary.styl
index d9a572d..5386e4c 100644
--- a/themes/aframe/source/css/secondary.styl
+++ b/themes/aframe/source/css/secondary.styl
@@ -36,10 +36,9 @@ a
     margin 0 1px
 
 .nav
-    border-top 1px dashed rgba(0,0,0,0.1)
-    border-bottom 1px dashed rgba(0,0,0,0.1)
-    margin-bottom 25px
-    padding 25px 0
+    border-top 1px dashed rgba(0,0,0,.1)
+    margin-bottom 15px
+    padding-top 25px
 
 .nav-link
     color rgba(0,0,0,0.35)
@@ -89,8 +88,18 @@ a
         line-height 1.6
 
 .sponsor
-    background-color:  #ef2d5e
-    position: relative    
+    border-bottom 1px dashed rgba(0,0,0,.1)
+    display block
+    margin 0 0 25px
+    padding-bottom 50px
+    position relative
+
+    a.mozilla-logo
+        background-color $color-primary
+
+    .message
+        margin 0
+        padding-bottom 25px
 
 .example__iframe
     height 100vh
```

File: themes/aframe/source/css/_common.styl [hunk 3]
```diff
@@ -226,3 +226,34 @@ h2, h3, h4
...
+  line-height: 30px;
+  vertical-align: top;
+  font-weight: 300;
+}
+a.mozilla-logo {
+  display: inline-block;
+  opacity: 0.8;
+  background: url(../images/mozilla-logo.svg) no-repeat;
+  height: 24px;
+  width: 88px;
```
Comment (cvan) [inline themes/aframe/source/css/_common.styl]: can you change this to `84px`? (it's the correct proportional width so the logo fits snuggly to the edges.)

File: themes/aframe/layout/partials/secondary/sidebar_header.ejs [hunk 4]
```diff
@@ -71,4 +71,14 @@ links = links.map(function (link) {
       </li>
     <% }) %>
   </ul>
+
+  <span class="sponsor" itemscope itemprop="organization" itemtype="http://schema.org/Organization">
```
Comment (cvan) [inline themes/aframe/layout/partials/secondary/sidebar_header.ejs]: we can add this to a separate `sponsor.ejs` partial file (I can do it in a follow-up pull request; no need to worry about it)

Comment (marti1125) [inline themes/aframe/layout/partials/secondary/sidebar_header.ejs]: @cvan I added your changes please review again :D

**Comment by cvan** [comment]
🎉 this looks great - many thanks for your contribution (and patience) 👍

**Comment by marti1125** [comment]
yeah!!

**Pull request merged**